packages/client/src/workspace/provider.ts

```diff
@@ -1,8 +1,7 @@
 import { init as workspacePlatformInit } from "@openfin/workspace-platform"
 
 import {
-  registerCreditAcceptedNotifications,
-  registerCreditQuoteNotifications,
+  registerCreditNotifications,
```

Comment:
note (x-ref with `Launcher/index.tsx`) .. both registrations .. so I think we need to our level Launcher up
(like the dup notifications issue, tho, we could do this as a next step ..)